src/Util/AdminObjectAclData.php

```diff
@@ -192,11 +176,11 @@ public function setAclUsersForm(FormInterface $form)
 
     /**
      * Gets ACL users form.
-     *
-     * @return FormInterface
      */
-    public function getAclUsersForm()
+    public function getAclUsersForm(): FormInterface
     {
+        \assert(null !== $this->aclUsersForm);
```

Comment:
Same, there is no argument for this assert ; so I would update the return type instead.